rts/Lua/LuaSyncedCtrl.cpp

```diff
@@ -6277,11 +6283,11 @@ int LuaSyncedCtrl::SetSmoothMesh(lua_State* L)
 
 /***
  * @function Spring.SetSmoothMeshFunc
- * @func lua_function
- * @param arg1?
- * @param arg2?
- * @param argn?
- * @return ?nil|number absTotalHeightMapAmountChanged
+ * @param lua_function function
+ * @param arg1 any?
+ * @param arg2 any?
+ * @param argn any?
```

Comment:
Here you can use the `...` param spec

Reply:
This is another example where I couldn't understand what was going on in the code so I didn't touch it. 

Note to self: `@return` is invalid here because it doesn't have a "name"

Reply:
The extra args are whatever the function from the 1st arg accepts and are just forwarded to it.

Reply:
Fixed in e05dce3cc96f887c76ae8032d6c5fca4708bdc4d